frontend/public/components/pod.jsx

```diff
@@ -193,6 +193,26 @@ const Details = ({obj: pod}) => {
 
     </div>
 
+    <div className="co-m-pane__body">
+      <h1 className="co-section-title">Init Containers</h1>
+      <div className="row">
+        <div className="co-m-table-grid co-m-table-grid--bordered">
+          <div className="row co-m-table-grid__head">
+            <div className="col-sm-2 col-xs-4">Name</div>
+            <div className="col-sm-2 hidden-xs">Id</div>
+            <div className="col-sm-2 col-xs-8">Image</div>
+            <div className="col-md-2 col-sm-2 hidden-xs">Security Scan</div>
+            <div className="col-md-1 col-sm-2 hidden-xs">State</div>
+            <div className="col-md-1 col-sm-2 hidden-xs">Restart Count</div>
+            <div className="col-md-2 hidden-sm hidden-xs">Started At</div>
+          </div>
+          <div className="co-m-table-grid__body">
+            {pod.spec.initContainers.map((c, i) => <ContainerRow key={i} pod={pod} container={c} />)}
```

Comment:
Not new, but `c.name` might be a better key (for both init containers and regular containers)

https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
On second thought, you can disregard this comment since the containers array is immutable.